docs/pages/access-controls/reference.mdx

```diff
@@ -1,101 +1,46 @@
 ---
-title: Access Controls Reference Documentation
-description: This reference shows you the configuration settings that you can include in a Teleport role, which enables you to apply access controls for your infrastructure.
-h1: Teleport Access Controls Reference
+title: Teleport Access Controls Reference
+description: Explains the configuration settings that you can include in a Teleport role, which enables you to apply access controls for your infrastructure.
+tocDepth: 3
 ---
 
 This guide shows you how to use Teleport roles to manage role-based access
 controls (RBAC) in your Teleport cluster.
 
-## Roles
-
-A Teleport `role` works by having two lists of rules: `allow` rules and `deny` rules.
-When declaring access rules, keep in mind the following:
+A Teleport role manages access by having two lists of rules: `allow` rules and
+`deny` rules.  When declaring access rules, keep in mind the following:
 
 - Nothing is allowed by default.
 - Deny rules get evaluated first and take priority.
 
-A rule consists of two parts: the resources and verbs. Here's an example of an
-`allow` rule describing a `list` verb applied to the SSH `sessions` resource.  It means "allow
-users of this role to see a list of active SSH sessions".
-
-```yaml
-allow:
-  - resources: [session]
-    verbs: [list]
-```
-
-If this rule was declared in the `deny` section of a role definition, it would
-prohibit users from getting a list of active sessions. You can see all of the
-available resources and verbs under the `allow` section in the example role
-configuration below.
-
-To manage cluster roles, a Teleport administrator can use the Web UI or the
-command line using [tctl resource commands](../reference/resources.mdx).
-To see the list of roles in a Teleport cluster, an administrator can execute:
+To manage cluster roles, as well as other dynamic resources, a Teleport
+administrator can use the Web UI or a [Teleport client
+tool](../management/dynamic-resources.mdx) like `tctl`, the Teleport Terraform
+provider, or the Teleport Kubernetes operator, as well as [custom API
+clients](../api/rbac.mdx).
```

Comment:
```suggestion
You can any of the following to manage cluster roles and other dynamic resources:

- Teleport Web UI
- [Teleport clienttool](../management/dynamic-resources.mdx) like `tctl`
- Teleport Terraform provider
- Teleport Kubernetes operator
- [custom API clients](../api/rbac.mdx)
```
I'm not sure this feels particularly natural here, but the two "as well as" clauses seemed a little tricky to pull off.